CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1462,9 +1463,6 @@ def build_workflow(subject_id, sub_dict, cfg, pipeline_name=None, num_ants_cores
 
     pipeline_blocks += [func_despike_template]
 
-    if "Template" in target_space_alff and target_space_nuis == "native":
-        pipeline_blocks += [warp_denoiseNofilt_to_T1template]
```

Comment:
I think this was added back in in error from merge conflict resolution in 66400a8006288f7c83d5457ccfd1f423686c2754